Header header
uint8 motor_id
bool connected
int32 rpm
float32 voltage
float32 current
float32 temperature
